observability/logging/logging-6.0/log6x-clf.adoc

```diff
@@ -0,0 +1,143 @@
+
+//AtLeastOncePersistent
+//NOTE: This option may be omitted in the first version, until we are satisfied there are use cases.
+
+Like AtLeastOnce, but forces the use of a persistent buffer, possibly in addition to acknowledgements. Using buffer and acknowledgements together gives better reliability than buffering alone.
+
+Use AtLeastOncePersistent when:
+
+You have a special situation where end-to-end acknowledgements alone do not work.
+You want to force the use of an extra-large buffer to work around large overload spikes.
+Note: Very large buffers cannot fix long-term overload, they can only work-around temporary spikes. The long-term average throughput must be within the collectors capacity, otherwise the system cannot "catch up". Once the buffer fills, logs will be lost as if the buffer wasn't there, and logs that are delivered will have high latency from waiting around in the buffer.
+
+Note: The log collector cannot guarantee fully reliable end-to-end delivery. It has no control over the reliability or throughput of sources and sinks. AtMostOnce takes advantage of reliability features at the source and sink, but the end-to-end result is only as good as the weakest link.
+
+compression: string
+Enable compression if supported
+
+Value indicates compression type (e.g. "zlib", "gzip"), valid values depend on the output type.
+Error if output does not support compression or does not recognize the value.
+maxWriteBytes: byte measurement (100, 1K etc.)
+Limits the maximum bytes of data that will be sent in a single remote "write" or "send" operation. Defaults are determined by the underlying sink.
+
+minRetryDuration`: duration (1s, 200ms)
+The first retry after a connection failure will happen after minRetryDuration The delay between subseqent retries will increase up to maxRetryDuration minDelay for the first attempt, delay increases up to maxDelay and then repeats.
+
+maxDelay: maximum delay between attempts.
+maxRetryDuration`: duration (1s, 200ms)
+See minRetryDuration.
+
+[id="log6x-CLF-filters_{context}"]
+== Filtering logs
+
+The following filters are available:
+// Should be an include for each, create: modules/log6<filtername>.adoc - Associated JIRA?
+detectMultilineException:: Detects and handles multiline exceptions.
+parse:: Parses log messages using Vector's parsing language.
+openshiftLabels:: Adds OpenShift-specific labels to log messages.
+
+// Needs to be validated by engineering for 6.0.
+include::modules/logging-audit-log-filtering.adoc[leveloffset=+1]
+
+// Needs to be validated by engineering for 6.0.
+include::modules/logging-content-filter-drop-records.adoc[leveloffset=+1]
+
+// Needs to be validated by engineering for 6.0.
+include::modules/logging-content-filter-prune-records.adoc[leveloffset=+1]
```

Comment:
@anpingli / @jcantrill : need to know if this can be used for 6.0 & if not what changes are needed.

Reply:
https://github.com/openshift/openshift-docs/blob/enterprise-4.16/modules/logging-content-filter-prune-records.adoc

Reply:
Created a https://github.com/openshift/openshift-docs/pull/78477 for verification of the CLF content.

Reply:
yes. we can use it.